lnbits/core/views/user_api.py

```diff
@@ -0,0 +1,86 @@
+from typing import Optional
+
+from fastapi import APIRouter, Body, Depends, Response, status
+from fastapi.responses import JSONResponse
+
+from ..crud import create_user, get_account_by_email, get_user
+from ..models import User, createUser
+from ..services import load_user, login_manager
+
+user_router = APIRouter()
+
+
+@user_router.get("/api/v1/user")
+async def user(user=Depends(login_manager)) -> User:
+    return user
+
+
+@user_router.post(
+    "/api/v1/login", description="Login to the API via the username and password"
+)
+async def login_endpoint(
+    response: Response,
+    username: str = Body(),
```

Comment:
- should be called `email`. See line 35: `user = await get_account_by_email(username)`

Reply:
my issue with email is, i dont want to expect an email from new users to create an account.